nbs/075_models.RNNAttention.ipynb

```diff
@@ -0,0 +1,540 @@
+{
```

Comment:
<p>You should replace by name by yours. It's your implementation. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/timeseriesAI/tsai/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Ok, I'll edit it.</p>